src/foam/dao/FileJournal.java

```diff
@@ -63,7 +63,7 @@ public void remove(FObject obj, Detachable sub) {
 
   public void remove(Object id, Detachable sub) {
     try {
-      bw.write("r({\"id\":" + id + "})");
+      bw.write("r({\"id\":\"" + id + "\"})");
```

Comment:
But if the id isn't a string, then this will be wrong.

JSON.stringify({id:'foo'});
"{"id":"foo"}"
JSON.stringify({id:42});
"{"id":42}"